scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -134,6 +139,11 @@ databaseChangeLog:
                   type: integer
                   constraints:
                     nullable: true
+              - column:
+                  name: guid
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
```

Comment:
Can we add these changes to a new changeset? Since a few of us have deployed a version internally and db reverts are not that easy anymore.